db/migrate/20171001035221_move_setting_to_tags.rb

```diff
@@ -0,0 +1,23 @@
+class MoveSettingToTags < ActiveRecord::Migration[5.0]
+  def up
+    print "\n"
+    Character.where.not(setting: nil).each do |character|
+      setting = Setting.where(name: character.setting).first
+      setting ||= Setting.create!(user: character.user, name: character.setting)
+      character.settings << setting
+      print "#{character.name} migrated\n"
```

Comment:
This looks like it's going to clutter the output A Lot and so far migrations don't do that (at least not a ton) – could we output `\r(#{index} / #{total_count})` instead, so it looks like a progress thing? (`\r` resets the line instead of doing a linebreak, then you'd presumably fetch and save `total_count` up the top and increment `index` as you go along, so it shows `(1/250)` then `(2/250)`…)